Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h

```diff
@@ -330,6 +330,10 @@ PREHEAT SETTINGS
 #define ASA_PREHEAT_HPB_TEMP 105
 #define ASA_PREHEAT_FAN_SPEED 0
 
+#define PC_PREHEAT_HOTEND_TEMP 275
+#define PC_PREHEAT_HPB_TEMP 105
+#define PC_PREHEAT_FAN_SPEED 0
```

Comment:
Why only here and not in all other variants?

Reply:
the other variants do not have the fan like this (this is MK2)

Reply:
And IT the `aaa_PREHEAT_FAN_SPEED` is nowhere used in the code.
So so it is very old.
BTW The MK2 RAMBo13a doesn't have that. guess someone forgot to cleanup the RAMBo10a